src/main/java/Duke.java

```diff
@@ -1,10 +1,287 @@
+import exceptions.DeadlineParamsFormatException;
+import exceptions.EventParamsFormatException;
+import exceptions.TaskIndexNotFoundException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    static ArrayList<Task> tasksList = new ArrayList<Task>();
+
+    public static void sendLogo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    public static void sendGreeting(){
+        String greeting_word="------------------------------------------------------------\n"
+            + "Hello! I'm miniJohn\n"
+            + "What can I do for you?\n"
+            + "------------------------------------------------------------";
+
+        System.out.println(greeting_word);
+    }
+
+    public static String getUserInput(){
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        return line;
+    }
+
+    public static String[] getCommandTypeAndParams(String userCommand){
+        String[] userCommandWords = userCommand.split(" ");
+        String commandType = userCommandWords[0];
+        String commandParams = null;
+        if(userCommandWords.length > 1){
+            commandParams = String.join(" ",Arrays.copyOfRange(userCommandWords,1,userCommandWords.length));
+        }
+        String[] commandTypeAndParams = {commandType, commandParams};
+        return commandTypeAndParams;
+    }
+
+    public static String executeListCommand(String commandParams) {
+        //[have not done]: exception
+        if(commandParams != null){
+            return "ListParamsError: List command do not have Params!";
+        }
+
+        String tasksListString = "";
+        for(int i = 0; i < tasksList.size() ; i++){
+            tasksListString += (i + 1) + "." + tasksList.get(i).toString() + "\n";
+        }
+        return tasksListString;
+    }
+
+    public static String executeAddTaskCommand(String newTaskString){
+        if(newTaskString == null)return "";
```

Comment:
Try to have wrapping curly braces even though it is only one line and put them on different lines